tests/integration_tests/functional/test_snapshot_phase1.py

```diff
@@ -36,20 +37,18 @@ def test_snapshot_phase1(
     vm.add_net_iface()
 
     static_cpu_template = None
-    cpu_template_name = "None"
     if isinstance(cpu_template_any, str):
         static_cpu_template = cpu_template_any
-        cpu_template_name = f"static_{cpu_template_any}"
     elif isinstance(cpu_template_any, dict):
         vm.api.cpu_config.put(**cpu_template_any["template"])
-        cpu_template_name = f"custom_{cpu_template_any['name']}"
```

Comment:
oh how I cannot wait for Firecracker 2.0 to clean this all up hahahahah

Reply:
+1,000,000,000